### ISSUE TITLE
[Request] IFTTT Integration

### ISSUE THREAD

**Issue by rodrigoswz** [wallabag/wallabag#3701]
Could you please create an API or something else so we can automate Wallabag with other services using IFTTT? This is something that makes me continue to use Instapaper/Pocket. Other than that, Wallabag is perfect for me.

I know there is already issue #3634 that looks like this request, but there the user asks for help for their (unsuccessful) attempt to interact with IFTTT. Here I am asking for an official solution or any other that actually works.

**Comment by foxmask** [comment]
If you want, You can do that with [Trigger Happy](https://github.com/push-things/django-th) instead of using IFTTT and post stuff from Wallabag to any of the [supported services](http://trigger-happy.readthedocs.io/en/latest/installation_guide/services.html#supported-services)

**Comment by j0k3r** [comment]
I can find where we already discussed about IFTTT integration.
As far as I remember an IFTTT integration cost money, a lot of money. And that's why there is no integration yet and I don't think there will be integration in the future.

**Comment by foxmask** [comment]
@j0k3r may be in #3634 with pocket

**Comment by j0k3r** [comment]
@foxmask nope, I think it was a (maybe) private conversation with @nicosomb after they contacted IFTTT.

**Comment by foxmask** [comment]
@j0k3r ok I see. At least, It remains the API of each project to make them speak to each other. That will cost just some elbow oil ;)

**Comment by rodrigoswz** [comment]
Yes, I know official integration costs money... What I wanted was an API to be used with Webhooks. As with many services like Join, Sleep As Android and Discord.

**Comment by j0k3r** [comment]
Oh I see, that's a good idea 👍

**Comment by ScrumpyJack** [comment]
Can we have node-red integration?

**Comment by j0k3r** [comment]
@ScrumpyJack feel free to work on it if you have time 👍
